config/app/main/general/templates.json

```diff
@@ -0,0 +1,22 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^5.0.0/components/context.jsonld",
+  "@graph": [
+    {
+      "comment": "Generic template engine handler, finds the appropriate template engine to use based on the template extension.",
```

Comment:
Due to the existence of the class `GenericTemplateEngine` I would not call this a generic template engine in the comment. Just something like "Template engine that find the ..."